demo/java/README.md

```diff
@@ -1,22 +1,75 @@
 # Usage
 
-**step 1.** Compile Utils with Java APIs.
+**step 1.** Check Java API classes existence.
+
+Please check the existence of `*.class` in `${MMDEPLOY_DIR}/csrc/mmdeploy/apis/java/mmdeploy`.
+
+If there is no existence of `*.class`, please follow this [tutorial](../../csrc/mmdeploy/apis/java/README.md) to build Java class.
+
+**step 2.** Install Apache Ant for building Java demo.
```

Comment:
I think it's better to organize it as follows:
1. Install Apache Ant
2. Build OpenCV jar
3. Build MMDeploy Java API
4. Build Demo

Reply:
P.S., Can opencv's prebuilt package be used instead of manually built it?
https://github.com/opencv/opencv/releases/download/4.7.0/opencv-4.7.0-android-sdk.zip